build.xml

```diff
@@ -2037,6 +2043,12 @@
         </classpath>
     </taskdef>
     <mkdir dir=".settings" />
+    <echo file=".settings/org.eclipse.jdt.core.prefs"><![CDATA[eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.ignoreUnnamedModuleForSplitPackage=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=11
```

Comment:
Coerces code to be compiled targeting java 11 (if default configured JVM is newer)